lib/encoding.js

```diff
@@ -6,12 +6,13 @@
  * @fileoverview Global |this| required for resolving indexes in node.
  * @suppress {globalThis}
  */
-if (typeof module !== "undefined" && module.exports &&
-    !this["encoding-indexes"]) {
-    require("./encoding-indexes.js");
-}
-
-(function(global) {
+(function(global, factory) {
```

Comment:
I wanted to preserve the scope of the original IIFE as it was for some odd reason. By moving the statement where the indexes are being loaded, it worked fine, so i'm going to push those changes. Thanks for the input!